cognite/client/_api/sequences.py

```diff
@@ -1009,72 +993,108 @@ def delete_range(self, start: int, end: int | None, id: int | None = None, exter
         sequence = self._cognite_client.sequences.retrieve(id=id, external_id=external_id)
         assert sequence is not None
         post_obj = Identifier.of_either(id, external_id).as_dict()
-        post_obj.update(self._process_columns(column_external_ids=[sequence.column_external_ids[0]]))
+        post_obj.update(self._wrap_columns(column_external_ids=sequence.column_external_ids))
         post_obj.update({"start": start, "end": end})
-        for data, _ in self._fetch_data(post_obj):
-            if data:
-                self.delete(rows=[r["rowNumber"] for r in data], external_id=external_id, id=id)
+        for resp in self._fetch_data(post_obj):
+            if rows := resp["rows"]:
+                self.delete(rows=[r["rowNumber"] for r in rows], external_id=external_id, id=id)
 
+    @overload
     def retrieve(
         self,
-        start: int,
-        end: int | None,
-        column_external_ids: SequenceType[str] | None = None,
-        external_id: str | SequenceType[str] | None = None,
-        id: int | SequenceType[int] | None = None,
+        id_or_external_id: int | str,
```

Comment:
Why this change? It's now inconsistent with the rest of the sdk, as well as other methods in this module.

Reply:
To reflect the API which is either `id` or `external_id`, I know this is no other place in the SDK, but I think `sequences.rows` is a bit unique. The advantage you get is that you get a proper overload implementation which is difficult when you split it into `id` and `external_id`.

Reply:
Is there another reason other than getting twice the amount of overloads?

Reply:
No, but I struggled to make it work with twice amount of overloads so if you have a solution I am all ears

Reply:
Done one fix.

Reply:
I added a suggestion to how it can be done now.

Reply:
Thank you